app/jobs/sync_salesforce_leads_job.rb

```diff
@@ -0,0 +1,28 @@
+class SyncSalesforceLeadsJob < ApplicationJob
+  queue_as :leads
+  sidekiq_options lock: :while_executing,
+                  on_conflict: :reject
+
+  def perform(uuid=nil)
+    if uuid
+      sf_leads = OpenStax::Salesforce::Remote::Lead.where(accounts_uuid:uuid)
+    else
+      sf_leads = OpenStax::Salesforce::Remote::Lead.all
+    end
+
+    total sf_contacts.count
+    processed = 0
+
+    sf_leads.each do |sf_lead|
+      Lead.cache_local(sf_lead)
+      processed += 1
+      total processed
```

Comment:
Other jobs for objects have  `at processed, "#{processed} processed"`.  Just pointing out the difference in case they should be the same.